simcon_project/users/views/assignment_management.py

```diff
@@ -90,11 +90,9 @@ def assignment_management_view(request):
     """
     Main view for assignment management. Has table of researcher's assignments with modals for more info and
     option to delete.
-    :param request:
-    :return:
     """
     assignment_rows = []
-    researcher = Researcher.objects.get(id=request.user.id)
+    researcher = Researcher.objects.filter(id=request.user.id).first()
```

Comment:
can user ids be non unique?

Reply:
Yeah this shouldn't ever fail, can't remember exactly why we changed it. User should be logged in at this point.